spec/lib/github_authentication/coverband_reports_web_spec.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
```

Comment:
I think this needs to be renamed to `coverband_reporters_web` right now it's reports